<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigo</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="newGameButton">
          <property name="text">
           <string>New game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="undoButton">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="passButton">
          <property name="text">
           <string>Pass</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scoreButton">
          <property name="text">
           <string>Score</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="estimateScoreButton">
          <property name="text">
           <string>Estimate score</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoMarkButton">
          <property name="text">
           <string>Mark dead stones</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="randomButton">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>AI tools:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="markSlider">
          <property name="toolTip">
           <string>Output target for training data.</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
          <property name="sliderPosition">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="markButton">
          <property name="toolTip">
           <string>Saves the current move as a training example.</string>
          </property>
          <property name="text">
           <string>Save move</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="laipButton">
          <property name="text">
           <string>Let AI play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="evalMoveButton">
          <property name="text">
           <string>Evaluate move</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="evaluationLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="plotAllEvaluationsButton">
          <property name="text">
           <string>Plot all evaluations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="examplesTrain">
          <property name="text">
           <string>Train on examples</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="simulationsTrain">
          <property name="text">
           <string>Train on simulations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reinitButton">
          <property name="text">
           <string>Reinitialize</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="text">
       <string>Score
Green: 
Blue:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
